spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/tracing/BraveAutoConfiguration.java

```diff
@@ -130,15 +131,22 @@ public brave.Tracer braveTracer(Tracing tracing) {
 	}
 
 	@Bean
+	@Scope("prototype")
 	@ConditionalOnMissingBean
-	public CurrentTraceContext braveCurrentTraceContext(List<CurrentTraceContext.ScopeDecorator> scopeDecorators,
+	public CurrentTraceContext.Builder braveCurrentTraceContextBuilder() {
```

Comment:
The tests should be updated to assert that this new bean is defined and that it backs off when a custom `CurrentTraceContext.Builder` bean is defined.